index.html

```diff
@@ -1049,6 +1049,395 @@ <h2>Evidence</h2>
+either including a relationship object in the subject's <a>claim</a>, or by issuing a separate relationship credential to the holder. Either way, the relationship object is used to
+describe the relationship between the holder and the subject of the claim. </p>
+
+<p>It is outside
+the scope of this standard how the issuer verifies the subject, the holder and the
+relationship between them.
+
+<p>
+Note: it is for further study which, if any, of the above mechanisms is to be 
+standardised by the working group.
+
+     </p>
+
+   <section>
+
+      <h3>The Relationship Object</h3>
+
+      <p>
+The Relationship object states the relationship between a subject, identified by an ID, and an object, also identified by an ID e.g. a value of "Parent" indicates that the subject is the parent of the object. 
+     </p>
+      <p>
+Note that values of the relationship type will be defined in other standards, 
+but it is 
+likely that typical values might be "Parent" (when the subject is a child) and 
+"Owner" (when the subject is a pet) etc.
+      </p>
+
+<p>
+Note that the following text and the two examples have not yet been finalised
+by the working group.
+</p>
+<p>
+When the relationship object is contained
+within a subject's claim, then the holder is the object of the
+relationship and the claim's subject is the subject of the relationship. When the
+relationship property is contained in the holder's relationship credential, the holder is the subject of both the relationship credential and the relationship, and the 
+relationship object refers to the subject of one or more claims or verifiable credentials that are held by the holder.
+
+      </p>
+ 
+     
+        <pre class="example nohighlight" title="An example of the relationship 
+property in a child's credential, so that a parent can hold it">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageUnder": 16,
+		"relationship": {
+			"type": "Child",
+			"id": "did:example:ebfeb1c276e12ec211f712ebc6f"
+		}
+	},
+	"proof": {
+		"type": "RsaSignature2018",
+		"created": "2017-06-17T10:03:48Z",
+		"creator": "https://dmv.example.gov/issuers/14/keys/234",
+		"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+		"signatureValue": "pY9...Cky6Ed = "
+	}
+} 
+     </pre>
+
+<pre class="example nohighlight" title="An example of the relationship 
+object in a relationship credential so that a parent can hold their child's
+verifiable credentials">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "RelationshipCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1c276e12ec211f712ebc6f",
+		"relationship": {
```

Comment:
"relationship" should call out the specific relationship, i.e. it should be something like "parent".

Reply:
@dlongley I will improve the description of the relationship object to make it even clearer,  viz:

The Relationship object states the relationship between the subject of the claim, identified by an ID, 
and this object, also identified by an ID e.g. a value of "Parent" indicates that the claim
subject is the parent of the relationship object.

So by using this definition, we can see that the relationship object is "did:example:ebfeb1c276e12ec211f712ebc6f",
thus this object is the child of the subject of the claim viz "did:example:ebfeb1f712ebc6f1c276e12ec21",